src/smartstream/examples/filter/src/lib.rs

```diff
@@ -0,0 +1,7 @@
+use fluvio_smartstream::{smartstream, SimpleRecord};
+
+#[smartstream(filter)]
+pub fn my_filter(record: &SimpleRecord) -> bool {
+    let value = String::from_utf8_lossy(record.value.as_ref());
```

Comment:
no need for intermediate value

Reply:
Do you mean not using String? Or not assigning 'value' and just doing everything inline?